microcosm_elasticsearch/store.py

```diff
@@ -44,10 +44,28 @@ def doc_type(self):
 
     @property
     def index_name(self):
+        # Deprecated - use get_index_name instead()
```

Comment:
How do we mark functions for deprecation in core libraries?

Reply:
You can use a deprecation warning https://docs.python.org/3/library/exceptions.html#DeprecationWarning, but it's not clear you want to, since it'll create a lot of warnings in Frigg (which we may not want to update to the latest code right now).